packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -2280,6 +2292,15 @@ function updateSuspenseComponent(current, workInProgress, renderLanes) {
           prevOffscreenState === null
             ? mountSuspenseOffscreenState(renderLanes)
             : updateSuspenseOffscreenState(prevOffscreenState, renderLanes);
+        if (enableTransitionTracing) {
+          const currentTransitions = getSuspendedTransitions();
+          if (currentTransitions !== null) {
+            const primaryChildUpdateQueue: OffscreenQueue = {
+              transitions: currentTransitions,
+            };
+            primaryChildFragment.updateQueue = primaryChildUpdateQueue;
```

Comment:
How does this get cleared? In the PR description it says that you'll set the `transitions` field to `null` but I don't see that.

I would null out the entire update queue field (`fiber.updateQueue = null`) instead

Reply:
I set it in the commit phase (which is a separate PR, though I can also combine them).